stable/heapster/templates/role.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.rbac.create -}}
+{{- if .Values.resizer.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: system:pod-nanny
```

Comment:
Let's make this `{{ template "fullname" . }}-pod-nanny` and rename the file to `pod-nanny-role.yaml`.